package.json

```diff
@@ -29,7 +29,7 @@
     "memory": 4096,
     "awsRole!important": "arn:aws:iam::282898975672:role/spacecat-role-services",
     "testUrl": "/_status_check/healthcheck.json",
-    "awsAttachAuthorizer": "spacecat-token-authorizer",
+    "awsAttachAuthorizer": "",
```

Comment:
this should still use the authorizer...

Reply:
it's only there for testing, will revert it before merging 👍